src/bindgen.rs

```diff
@@ -181,7 +185,7 @@ pub fn wasm_bindgen_build(
         .arg(dts_arg)
         .arg(target_arg);
 
-    if debug {
+    if profile.debug_info_is_enabled() {
```

Comment:
This may not be quite the right question to ask here because `--debug` may run the risk of significantly slowing down the bindings. (in that this is enabled for profiling mode I think?)